src/game/demo.c

```diff
@@ -22,7 +22,7 @@ static uint32_t *m_DemoPtr = NULL;
 int32_t StartDemo(void)
 {
     TEXTSTRING *txt;
-    START_INFO start, *s;
+    RESUME_INFO start, *s;
```

Comment:
Triple check if this should load from `start` or `current`, IMO start is OK.

Reply:
Hm so `s = &g_GameInfo.start[m_DemoLevel];`. This should be updated to `current` maybe? Since the old start is now current? I guess we can test it when we get to it.